{"uuid":"3e99b7a1-d0b4-4442-945d-6adaab023cc1","historyId":"e17548b4dac56d21230b6f91fb57b70d","fullName":"com.RSstajyer.TestApi.apiTesting","labels":[{"name":"package","value":"com.RSstajyer.TestApi"},{"name":"testClass","value":"com.RSstajyer.TestApi"},{"name":"testMethod","value":"apiTesting"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"DoganEnes2"},{"name":"subSuite","value":"com.RSstajyer.TestApi"},{"name":"host","value":"IDB-25390"},{"name":"thread","value":"8060@IDB-25390.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"Performing a simple test using Rest-Assured"},{"name":"feature","value":"Performing different API Tests using Rest-Assured"},{"name":"epic","value":"Rest Assured POC - Example Tests with Stajyer"},{"name":"severity","value":"minor"}],"links":[],"name":"apiTesting","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Çarpma sonucu yanlış expected [15] but found [25]","trace":"java.lang.AssertionError: Çarpma sonucu yanlış expected [15] but found [25]\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1590)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:150)\r\n\tat org.testng.Assert.assertEquals(Assert.java:132)\r\n\tat org.testng.Assert.assertEquals(Assert.java:1431)\r\n\tat org.testng.Assert.assertEquals(Assert.java:1395)\r\n\tat com.RSstajyer.TestApi.apiTesting(TestApi.java:48)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:741)\r\n\tat org.testng.TestRunner.run(TestRunner.java:616)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"},"stage":"finished","description":"Example Test for executing GET request using rest assured specification - İki sayının çarpımı Dogan_Dagistanli","steps":[],"attachments":[],"parameters":[],"start":1753430267386,"stop":1753430270910}